sdk/core/core-rest-pipeline/samples-dev/node-sample.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary This samples shows how to make a GET request using default http client.
+ */
+import {
+  createDefaultHttpClient,
+  createHttpHeaders,
+  createPipelineRequest
+} from "@azure/core-rest-pipeline";
+
+async function main() {
+  const client = createDefaultHttpClient();
+  const stamp = Date.now();
+  const request = createPipelineRequest({
+    url: `http://example.com/?_=${stamp}`,
```

Comment:
I wonder if there is an unauthenticated endpoint we can hit on some Azure service to make this a real example. I was using Search's public data set but recently there was brouhaha about the public credentials

Reply:
search is the only one I knew. Yes it would be better to have some real input and output.  Storage SAS url is another thing we can make requests against, although it still requires users to generate it via Azure portal or some other ways.